sdk/appconfiguration/tests.native.yml

```diff
@@ -23,9 +23,8 @@ stages:
       TimeoutInMinutes: 90
```

Comment:
Do you know that context of tests.native? What is the purpose of this over the other test pipelines?

Reply:
This performs a GraalVM build, @srnagar would know more